examples/data-sources/kestra_namespace_file/data-source.tf

```
@@ -0,0 +1,4 @@
+data "kestra_namespace_file" "example" {
+  namespace_ = "io.kestra.mynamespace"
+  filename   = "my-file.yml"
```

```suggestion
  filename   = "myscript.py"
```
optional, to show that this is meant more for script deployments than for flow deployments